src/Tree/Tree.js

```diff
@@ -146,7 +146,7 @@ export class Tree extends Component {
                               role='treeitem'>
                               <div className='fd-tree__row'>
                                   {parent}
-                                  {<Dropdown isContextual />}
+                                  {<Dropdown />}
```

Comment:
did we ever get an answer on this `isContexual` prop?

Reply:
Yes, @chrismanciero found a commit where it was removed from `Dropdown`, but apparently just missed in a few of the places that called it.